modules/ROOT/pages/introduction/introduction-to-anypoint-connectors.adoc

```diff
@@ -0,0 +1,113 @@
+= Introduction to Anypoint Connectors
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Anypoint Connectors are reusable extensions of Mule that enable you to integrate a Mule app with third-party APIs and standard integration protocols. All connectivity in Mule 4 is provided through connectors.
+
+Using connectors in an app provides provides the following advantages:
+
+* Simplifies app development and saves time, since you don't have to program directly to APIs
+* Can be updated without requiring updates to other parts of the app
+* *Reviewers, are there other advantages I can add here?*
```

Comment:
Maybe also this one:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/23425672/70911078-68065600-2011-11ea-986b-132760371fce.png">